command/v7/labels_command.go

```diff
@@ -130,6 +138,22 @@ func (cmd LabelsCommand) fetchSpaceLabels(username string) (map[string]types.Nul
 	return cmd.Actor.GetSpaceLabels(cmd.RequiredArgs.ResourceName, cmd.Config.TargetedOrganization().GUID)
 }
 
+func (cmd LabelsCommand) fetchBuildpackLabels(username string) (map[string]types.NullString, v7action.Warnings, error) {
+	err := cmd.SharedActor.CheckTarget(false, false)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	cmd.UI.DisplayTextWithFlavor("Getting labels for buildpack {{.BuildpackName}} as {{.Username}}...", map[string]interface{}{
```

Comment:
It looks like there was a request on the story (after this PR was made) to include the stack name in the message shown to the user for consistency with other commands:
https://www.pivotaltracker.com/story/show/166986428/comments/204879378

Reply:
The PR is still open; we can do that